scripts/component_integration_tests.py

```diff
@@ -94,6 +95,14 @@ def main() -> None:
                 "namespace": "torchx-dev",
             },
         },
+        "kueue_job": {
```

Comment:
can we add an integration test that uses minikube similar to the existing one for KFP? 

ex: https://github.com/pytorch/torchx/pull/696